### DISCUSSION TITLE
docs: update usage section in readme

### DISCUSSION THREAD

**Discussion by Ananya2001-an** [avajs/ava#3178] [Ideas]
Please provide details about:

* What you're trying to do
Currently in the usage section of readme we have told users to run `npm init ava` to get started with ava but it's a bit misleading since someone may not have created a node project yet and might directly run this command thinking that it would create a node project with ava as a dependency in it which in fact doesn't happen. Also after creating the node project and running this command will not add the line `"type": "module"` in the package.json file by default so we need to tell users to add it explicitly.

I think we should update the readme docs with this extra information for clarity on steps that users should follow.

![image](https://user-images.githubusercontent.com/55504616/221794946-f6d22ec3-5383-4bd4-b58b-daac3ee12a85.png)

**Comment by Ananya2001-an** [comment]
I would like to work on this if it's approved.

**Reply by novemberborn** [reply]
Happy to take a PR with improvements — though I would expect a certain baseline understanding of Node and packages that we don't need to repeat in our documentation.

**Reply by Ananya2001-an** [reply]
> Happy to take a PR with improvements — though I would expect a certain baseline understanding of Node and packages that we don't need to repeat in our documentation.

We don't need to mention any packages just a line before `npm init ava` that the user must have an existing package.json file. And later mention to include the type: module line as well.
